docs/wsrep-system-index.md

```diff
@@ -3,6 +3,17 @@
 Percona XtraDB Cluster introduces a number of MySQL system variables
 related to write-set replication.
 
+### `log_query_errors`
+
+| Option         | Description            |
+| -------------- | ---------------------- |
+| Command Line:  | ``--log_query_errors`` |
```

Comment:
At present `--log_query_errors` is neither parsed by the command line nor from the config file. So, we should document that this variable can be set and updated only through client via SET GLOBAL/SESSION.

Reply:
updated